From 001e8442fbddf458b66704c296d54d503e352c18 Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Thu, 4 May 2017 18:09:10 +0300
Subject: [PATCH 1614/2241] arm64: defconfig: enable generic CPU cooling
 support (CPU_THERMAL)

Enable generic cpu cooling mechanism through CPU frequency reduction,
using thermal-zones and cooling device binding of thermal sensor
to CPU clusters.

Change-Id: I17e18ee860986cb2eca4d21cbbe53ec3c994cdfd
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/39154
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 5079255..15235cd 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -169,6 +169,7 @@ CONFIG_POWER_RESET_XGENE=y
 CONFIG_POWER_RESET_SYSCON=y
 # CONFIG_HWMON is not set
 CONFIG_THERMAL=y
+CONFIG_CPU_THERMAL=y
 CONFIG_ARMADA_THERMAL=y
 CONFIG_WATCHDOG=y
 CONFIG_ARMADA3700_WATCHDOG=y
-- 
2.7.4

